foundry.toml

```diff
@@ -0,0 +1,15 @@
+[profile.default]
+via_ir = false
+src = 'contracts'
+out = 'foundry_out'
+libs = ["foundry_libs"]
+test = 'foundry_test'
```

Comment:
Why use the prefix?

Reply:
we may use multiple test frameworks to differ from one another